loom/engine/loom2d/l2dStage.cpp

```diff
@@ -33,7 +33,7 @@ namespace Loom2D
 
 Stage *Stage::smMainStage = NULL;
 NativeDelegate Stage::_RenderStageDelegate;
-bool Stage::sizeDirty = true;
+bool Stage::sizeDirty = false;
```

Comment:
It's because the initial sizing and stuff is now handled in `core.cpp`, so window size doesn't need to be set initially (and in fact can break options such as `maximized`).